src/Plugin/DomainGroupSettings/ThemeSettings.php

```diff
@@ -0,0 +1,197 @@
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+    $this->configFactory = $config_factory;
+    $this->domainStorage = $domain_storage;
+    $this->languageManager = $language_manager;
+    $this->themeHandler = $theme_handler;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public static function create(ContainerInterface $container, array $configuration, $plugin_id, $plugin_definition) {
+    return new static(
+      $configuration,
+      $plugin_id,
+      $plugin_definition,
+      $container->get('config.factory'),
+      $container->get('entity_type.manager')->getStorage('domain'),
+      $container->get('language_manager'),
+      $container->get('theme_handler')
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function access(GroupInterface $group, AccountInterface $account) {
+    return AccessResult::allowedIfHasPermission($account, 'set localgov microsite theme override');
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildConfigurationForm(array $form, FormStateInterface $form_state, GroupInterface $group) {
+    $config_override = NULL;
+    if (!$group->isNew() && $domain = $this->domainStorage->load('group_' . $group->id())) {
+      $config_override = $this->loadConfigOverride($domain);
+      if ($config_override->isNew()) {
+        $config_override = NULL;
+      }
+    }
+
+    $site_config = $this->configFactory->get('system.theme');
+    $site_default = $site_config->get('default');
+    $site_admin = $site_config->get('admin');
+
+    $admin_options = ['' => ''];
+    $default_options = ['' => ''];
+    // Get all available themes.
+    $themes = $this->themeHandler->rebuildThemeData();
+    // Remove obsolete themes.
+    $themes = array_filter($themes, function ($theme) {
+      return !$theme->isObsolete();
+    });
+    uasort($themes, [ThemeExtensionList::class, 'sortByName']);
+    foreach ($themes as &$theme) {
+      if (!empty($theme->status)) {
+        if ($theme->getName() != $site_default) {
+          $default_options[$theme->getName()] = $theme->info['name'];
+        }
+        if ($theme->getName() != $site_admin) {
+          $admin_options[$theme->getName()] = $theme->info['name'];
+        }
+      }
+    }
+
+    $admin = $config_override ? $config_override->get('admin') : '';
+    $form['admin'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Admin theme'),
+      '#default_value' => $admin,
+      '#description' => $this->t("Override theme used for administration pages."),
+      '#options' => $admin_options,
+    ];
+    $default = $config_override ? $config_override->get('default') : '';
+    $form['default'] = [
+      '#type' => 'select',
+      '#title' => $this->t('Default theme'),
+      '#default_value' => $default,
+      '#description' => $this->t("Override theme used for user pages."),
+      '#options' => $default_options,
```

Comment:
LocalGov Microsite Base doesn't appear as an option, can it please?

Reply:
The options don't show the existing default. So you don't override the selection with the value it would be anyway.

I had wondered if I made the present blank option called: 'No override (Existing theme name)'

Reply:
Yes, that would work. I can imagine someone finding the options and playing around with it and not knowing how to put it back.